config/mediawiki/LocalSettings.php

```diff
@@ -139,3 +143,50 @@
 ## Features
 
 $wgRCWatchCategoryMembership = true;
+
+## added for AdvancedSearch (not cleared as general purpose, yet)
+
+if ( isMounted( 'skins/Vector' ) ) {
+	wfLoadSkin( 'Vector' );
+}
+
+if ( isMounted( 'extensions/Elastica' ) ) {
```

Comment:
Seems like these should be placed in the user's own LocalSettings.php file. Not sure why the'd be here. Would make it more difficult to manage and maintain. The browser tests discover automatically what needs to run, so there should be no need to auto-load them here. If you're interested in any extension specifically, it can be installed and enabled just like any other. And if not, the tests will also run without them.